content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+
+If you're doing a side-by-side migration, edit the `agent-port` configuration flag so it doesn't conflict with your Sensu 1.x client port.
+
+### 2. Log in to the Sensu web UI
+
+The [Sensu Go web UI][13] provides a unified view of your monitoring events with user-friendly tools to reduce alert fatigue and manage your Sensu instance.
+After starting the Sensu backend, open the web UI by visiting http://localhost:3000.
+You may need to replace `localhost` with the
+hostname or IP address where the Sensu backend is running.
+
+To log in, enter your Sensu user credentials, or use Sensu's default admin credentials (username: `admin` and password: `P@ssw0rd!`).
+Select the ☰ icon to explore the web UI.
+
+### 3. Install sensuctl on your workstation
+
+Sensuctl is a command line tool for managing resources within Sensu. It works by calling Sensu’s HTTP API to create, read, update, and delete resources, events, and entities. Sensuctl is available for Linux, Windows, and macOS.
+See the [installation guide][54] to install and configure sensuctl.
+
+### 4. Set up Sensu users
+
+Role-based access control (RBAC) is a built-in feature of Sensu Go.
+RBAC allows management and access of users and resources based on namespaces, groups, roles, and bindings.
+To learn more about setting up RBAC in Sensu Go, see the [RBAC reference][38] and the [guide to creating a read-only user][39].
+
+In addition to built-in RBAC, Sensu includes [license-activated][40] support for authentication using Microsoft Active Directory and standards-compliant Lightweight Directory Access Protocol tools like OpenLDAP.
+
+### 5. Install agents
+
+“Clients” are now represented within Sensu Go as abstract “entities” that can describe a wider range of system components (network gear, web server, cloud resource, etc.)
+Entities include “agent entities” (entities running a Sensu agent) and familiar “proxy entities”.
+
+The Sensu agent is available for Ubuntu/Debian, RHEL/CentOS, Windows, and Docker.
+See the [installation guide][53] to install, configure, and start Sensu agents.
+
+Make sure to include the localhost subscription by uncommenting the subscription section and adding `localhost` to the list of subscriptions, or by using the `--subscriptions localhost` flag.
+
+{{< highlight yml >}}
+##
+# agent configuration
+##
+...
+subscriptions:
+ - "localhost"
+...
+{{< /highlight >}}
+
+Now you should have Sensu installed and functional. The next step is to translate your Sensu 1.x configs to Sensu Go.
+
+## Translate your configuration
+
+The [Sensu translator][18] is a command-line tool to help you transfer your Sensu Core 1.x checks, handlers, and mutators to Sensu Go.
+
+### 1. Run the translator
+
+Before we start, output your Sensu 1.x configuration to use as a reference.
+
+{{< highlight shell >}}
+# Requires curl and jq
+curl -s http://127.0.0.1:4567/settings | jq . > sensu_config_original.json

```

Comment:
I'm not sure why this step is there. It's a nice idea as it captures the current settings of the sensu core install as seen back the api server..but it's necesary. Maybe provide a rationale for what this step provides?

Reply:
👍 I'll move this into a NOTE callout.